.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
```

Comment:
This looks nicely declarative, what does it do?

Reply:
I think version 2 is the "native to github" version of Dependabot, rather than the only semi-third party thing.